app-admin/cdist/files/test.patch

```diff
@@ -0,0 +1,45 @@
+From 3a321469a8ba5aea55220bd70bd4900de732e917 Mon Sep 17 00:00:00 2001
```

Comment:
Please always include a link to any relevant Gentoo bugs at the top of the patch as well as a link to where it came from.

Reply:
Ok, do it this evening :)